app/views/claims/_additional_respondents.html.slim

```diff
@@ -20,7 +20,7 @@
           = builder.revealed_content :no_acas_number_reason, values: [:interim_relief], classes: ['form-hint'], tag: 'span'
             = t 'claims.respondent.no_acas_number_note_two'
         = builder.revealed_content :has_acas_number, values: [:true]
-          = builder.text_field :acas_early_conciliation_certificate_number
+          = builder.text_field :acas_early_conciliation_certificate_number, hint: { text: t('claims.respondent.acas_early_conciliation_certificate_number.hint', path:guide_path).html_safe }
```

Comment:
@kalvinSolirius did you try to use _html on the translation string instead of html_safe?